.github/dependabot.yml

```diff
@@ -0,0 +1,27 @@
+- package-ecosystem: composer
+  directory: "/php/"
+  schedule:
+    interval: weekly
+    day: saturday
+    time: "03:00"
+    timezone: Europe/Paris
+  open-pull-requests-limit: 10
+  labels:
+  - 3. to review
+  - dependencies
+  allow:
+  - dependency-type: direct
+  - dependency-type: indirect
+- package-ecosystem: composer
+  directory: "/php/vendor/"
+  schedule:
+      interval: weekly
+      day: saturday
+      time: "03:00"
+      timezone: Europe/Paris
+  open-pull-requests-limit: 10
+  labels:
+      - 3. to review
+      - dependencies
+  allow:
+      - dependency-type: direct
```

Comment:
What is the purpose of the second block? The first one should pick up the composer.json file in /php already

Reply:
okay nice, didn't know that. Thought we need that since the dependencies are currently stored in this repo, too. (look into the vendor directory)

Reply:
Right, that won't work to my knowledge. The vendor directory would always need to be updated manually or maybe through a github action. 

https://github.com/dependabot/dependabot-core/issues/386 was the feature request before dependabot was acquired by github. We have the same with the 3rdparty repo

Reply:
should be addressed with https://github.com/nextcloud/all-in-one/pull/93/commits/ce25a495927ee681fee92c6157418d6b98f702d1 :)